apps/site/util/authorUtils.ts

```diff
@@ -0,0 +1,63 @@
+import authors from '@/authors.json';
```

Comment:
The problem that authors are missing from the `authors.json` comes from here, the functions don't differentiate between a frontmatter data that comes from the learn or the blog. As mentioned in the issue, the authors in the frontmatter of the learn section are valid, so we can just put their profile image and redirect to their GitHub user. But in the case of the blog it's more complex, because the frontmatter comes from the old website which just displayed the authors as such, which is why there's this intermediate json file which allows you to display an image and a correct name.

Reply:
It made sense to me to read the people from the `authors.json` file while doing the development;

1. We can show the full name and GitHub known user name at the same time for the contributors
2. Everyone can contribute but I am not sure if every contributor would accept a redirect link to their GitHub profile 🤔 
3. It allows us to redirect from blog listing/detail pages as well

Maybe Im overthinking 😄

Reply:
for profile pictures it's easier GitHub. For links from a website other than GitHub, the problem is that if you have 0 hands on it, there could be content that doesn't follow our code of conduct.
What's more, if the author(s) don't want to be linked on the page, they won't put themselves in the frontmatter.

Reply:
Makes sense, thanks! I added GitHub redirect links by default for authors that are not in `authors.json` 👍 I think we can leave it as is, for now, I will look into improving it with GitHub APIs in the following PR